cmake/modules/AddSwift.cmake

```diff
@@ -1833,7 +1840,21 @@ function(_add_swift_executable_single name)
       BINARY_DIR ${SWIFT_RUNTIME_OUTPUT_INTDIR}
       LIBRARY_DIR ${SWIFT_LIBRARY_OUTPUT_INTDIR})
 
+  # On Windows, libraries are prefixed with "lib", so we need to adjust FAT_LIBRARIES for this.
```

Comment:
The output name should be `swiftReflection.lib` not `libSwiftReflection.lib`.  Windows doesn't use the `lib` prefix on most libraries.